hooks/hooks.plugin.js

```diff
@@ -1,12 +1,19 @@
 import fp from 'fastify-plugin';
 import { AuthHandlersService } from './auth-handlers.service.js';
+import { PostOwnershipChecker } from './post-handlers.service.js';
 
 export default fp(async fastify => {
   const service = new AuthHandlersService(fastify.jwt);
```

Comment:
BTW, you can change AuthHandlersService instance naming, because this is't only one service anymore